templates/web/westminster/report/form/user.html

```diff
@@ -25,6 +55,9 @@ <h2 class="form-section-heading form-section-heading--private hidden-nojs">
     </button>
   [% END %]
     <button type="button" class="btn btn--block hidden-nojs js-new-report-user-show">Confirm by email</button>
+    <small id="or">or</small>
```

Comment:
Should this not be outside the `social_auth_enabled` IF?

Reply:
It will always be enabled there due to MyWestminster, but yes I guess so to match.